server/service/src/sync/translations/name.rs

```diff
@@ -156,6 +158,23 @@ impl SyncTranslation for NameTranslation {
         Some(ChangelogTableName::Name)
     }
 
+    fn should_translate_to_sync_record(
+        &self,
+        row: &ChangelogRow,
+        r#type: &ToSyncRecordTranslationType,
+    ) -> bool {
+        match r#type {
+            // By default will assume records needs to be pushed to central if change_log_type is implemented
+            ToSyncRecordTranslationType::PushToLegacyCentral => {
+                self.change_log_type().as_ref() == Some(&row.table_name)
+            }
+            ToSyncRecordTranslationType::PushToOmSupplyCentral => {
+                self.change_log_type().as_ref() == Some(&row.table_name)
+            }
```

Comment:
so syncing name to COMS as well, so don't need to remove name_id constraint on name link (yay)

but name and name_store_join being added to central as part of sync, so wouldn't sync back to OG (i.e. OG won't know COMS can now see this patient?) any painful patient merge shenanigans we need to consider here?

let's chat when you're in/free 🙏

Reply:
After our discussion, we need to confirm merging and vaccination card editability + other use cases. Apart from that integration tests pass, and ready to review, thanks : )